website/src/categories/whats-new.md

```diff
@@ -14,12 +14,51 @@ eleventyNavigation:
 </div>
 </header>
 
-
 <section class="ds-subpage-section">
 
 <div class="ds-subpage-section__wrapper">
 <section>
   
+## Accessibility Audit, new components and further updates
+<small>December 2024</small>
+
+### Accessibility Audit
+Accessibility is a core focus for us, and we recently conducted an accessibility audit with an external agency to evaluate all our components. We're pleased to report that only a few accessibility issues were identified during the audit. You can view the complete audit overview [here](https://github.com/orgs/sl-design-system/projects/2/views/63). Our goal is to resolve all identified issues by January 2025.
+
+### Timeline New Components
+Last period, we did a survey where all products using the SL Design System participated to figure out which components should have the highest priority. In the survey, we showed all requested components and asked about their UX impact, how high they are on the roadmap, and what the requirements are. We used the 'RICE' framework to calculate the priority for each request. Turns out, the Number Field, Tree View, Date Picker, and Time Picker are the big winners.
+
+[Here](https://github.com/orgs/sl-design-system/projects/5) you can find the timeline for the design and development of the new components.
+
+### Contextual Tokens and Unified Figma Library
+The past period, the team has been working hard on contextual tokens. Right now, the SL Design System is using component-specific tokens. By making our tokens contextual instead of component-specific, we’re improving UI consistency across all products. 
+
+We’re currently refactoring our components in code to use the new contextual token setup. At the same time, there are some big changes coming in Figma. Right now, every product has its own Figma library with SL Design System components. The goal is to have a single Figma library with one theme that works for all products. Designers will be able to switch between themes directly in this library using Figma Variables.
+
+In the end, this will make the SL Design System team more efficient—less maintenance and more time to focus on new components, patterns, and enhancements. For products, it means more consistent UI design overall.
+
+### New Components
+- [Paginator](https://storybook.sanomalearning.design/?path=/story/navigation-paginator--all)
```

Comment:
Can you please add a badge here with component status (`draft`), just like we had it previously added in older months :) ?